components/interface/VFBTermInfo/VFBTermInfo.js

```diff
@@ -562,6 +601,10 @@ export default class VFBTermInfoWidget extends React.Component {
         path = instanceID;
       }
     }
+    if (path.indexOf(GRAPHS) === 0 ) {
+      this.props.vfbGraph(SHOW_GRAPH, path.split(',')[1], path.split(',')[2]);
```

Comment:
@jrmartin could you add some info in a comment regarding what is path and what are we splitting please? Thx

Reply:
as discussed, the dispatch function should we passed through a redux connector component rather than as prop from the parent.
(reference to get the ref of the connected component in the application.js->componentDidMount here https://github.com/openworm/geppetto-application/pull/52/files)